### ISSUE TITLE
svgz not recognized in html output

### ISSUE THREAD

**Issue by nowox** [sphinx-doc/sphinx#3128]
I have my include as `.. figure:: foo.*` which I want to be expanded to "svgz", but it does not.  I got the error:

```
None:None: WARNING: no matching candidate for image URI u'foo.*' 
```

However on `visit_image` in `sphinx/writers/html.py` I have this: 

```
if uri.lower().endswith('svg') or uri.lower().endswith('svgz'):
```

From the mime_suffixes in `sphinx/util/images.py` the `svgz` extension is missing: 

```
mime_suffixes = {
    '.pdf': 'application/pdf',
    '.svg': 'image/svg+xml',
}
```

**Comment by tk0miya** [comment]
I don't know this is a bug or feature enhancement.
But certainly it is very inconvenience if we can't use `.svgz` files to images.

So I'd like to add `.svgz` support into 1.5-release branch as a bug.
@shimizukawa how do you think?

Note:
W3C says mime-type for `.svgz` is "image/svg+xml".
https://www.w3.org/TR/SVG/intro.html#MIMEType

And the our documentation also says HTML builders supports "image/svg+xml".
>supported_image_types = ['image/svg+xml', 'image/png', 'image/gif', 'image/jpeg']
http://www.sphinx-doc.org/en/1.4.8/builders.html#sphinx.builders.html.StandaloneHTMLBuilder

It's not too much to say that this is a bug :-)

**Comment by shimizukawa** [comment]
+1 for including to 1.5-release

**Comment by tk0miya** [comment]
Fixed at b799249.
Thank you for reporting
